app/main.js

```diff
@@ -47,7 +47,9 @@ function forceShowWindow() {
 
 function createWindow() {
   // Create the browser window.
-  win = new BrowserWindow({width: 500, height: 350});
+  var w = 1500;
```

Comment:
Was this intentional? I think a little bigger is OK, but 1500x1000 seems way too big by default to me.

Reply:
do you know how to do something like
`var w = window.innerWidth * 0.75;`

window is undefined

Reply:
Like 75% of the screen?  I'd prefer to just leave the window size as is, people can always resize it - maybe you could look into saving the size of the screen between instances in another PR.